assets/css/style.css

```diff
@@ -0,0 +1,86 @@
+h1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #227452;
+}
+
+p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 2em 24px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.headline {
+  background-color: #fff;
+  background-image: url("../../assets/img/background-image-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.logo-placeholder {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+.toolbar {
+  padding: 0 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+.menu {
+  padding: 1em;
+  background: none;
+  border: 10px;
+}
+
+.headline button {
+  display: flex;
+  flex-direction: row;
+  background: none;
+  align-items: flex-start;
+  padding: 8px;
```

Comment:
- [ ] I think there is more distance between the social icons and the button, please 🙏🏽 correct the distance to look similar to the Figma template.

| Figma template | Your template |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/212431402-21645534-12ea-4c6a-a061-3f89d4c28d01.png) | ![image](https://user-images.githubusercontent.com/58642949/212431444-89127888-46d7-4cbb-9ffe-7bcfc9f396d3.png) |